zephyr/CMakeLists.txt

```diff
@@ -120,4 +121,5 @@ zephyr_library_sources(
   ${ZEPHYR_CURRENT_MODULE_DIR}/rmw_microxrcedds_c/src/rmw_microros/time_sync.c
   ${ZEPHYR_CURRENT_MODULE_DIR}/rmw_microxrcedds_c/src/rmw_microros/ping.c
   ${ZEPHYR_CURRENT_MODULE_DIR}/rmw_microxrcedds_c/src/rmw_microros/timing.c
+  $<$<BOOL:${RMW_UXRCE_TRANSPORT_CUSTOM}>:${ZEPHYR_CURRENT_MODULE_DIR}/rmw_microxrcedds_c/src/rmw_microros/custom_transport.c>
```

Comment:
Please make it Kconfig.

Reply:
Given that the transport is hardwired at the moment. I'm tempted to simply remove the conditional from this line, and also remove the unused lines 62,63 (setting UDP addr and port).

Once we have a good pattern for converting between zephyr Kconfig and ros cmake config, we can do it properly.

Reply:
love that idea.

Reply:
done